### ISSUE TITLE
Test that minimalkv works

### ISSUE THREAD

**Issue by jtilly** [conda-forge/minimalkv-feedstock#16]
### Comment:

Can we add a test to the recipe that detects breakage such as we had with version 1.4.0?

A test could look like this:

```
test:
  - imports:
    - minimalkv
  commands:
    - pip check
    - python -c 'from minimalkv import get_store; get_store("fs", path=".")'
    - python -c 'from minimalkv import get_store; get_store("hfs", path=".")'
  requires:
    - pip
    - fsspec
```

However, this would mean we wouldn't test if `import minimalkv` works when `fsspec` is _not_ installed. Is there a way to have two separate tests in a recipe?

**Comment by xhochy** [comment]
> Is there a way to have two separate tests in a recipe?

Sadly not. It would be better to test that in CI of minimalkv directly.

**Comment by xhochy** [comment]
Shouldn't the above tests work without `fsspec`?

**Comment by jtilly** [comment]
The `fs` test works without `fsspec`. The `hfs` test doesn't. The second test would have detected the breakage in 1.4.0.

```
|/tmp/minimalkv|jtilly@XPS13 tmp → python -c 'from minimalkv import get_store; get_store("hfs", path=".")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/_get_store.py", line 145, in get_store
    store = create_store(type, params)
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/_store_creation.py", line 23, in create_store
    return _create_store_hfs(type, params)
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/_store_creation.py", line 114, in _create_store_hfs
    from minimalkv._hstores import HFilesystemStore
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/_hstores.py", line 9, in <module>
    from minimalkv.net.gcstore import GoogleCloudStore
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/net/gcstore.py", line 5, in <module>
    from minimalkv.fsspecstore import FSSpecStore, FSSpecStoreEntry
  File "/tmp/minimalkv/lib/python3.10/site-packages/minimalkv/fsspecstore.py", line 7, in <module>
    from fsspec import AbstractFileSystem
ModuleNotFoundError: No module named 'fsspec'
```

**Comment by jtilly** [comment]
> Sadly not. It would be better to test that in CI of minimalkv directly.

Addressing this in https://github.com/data-engineering-collective/minimalkv/pull/57.

**Comment by xhochy** [comment]
Reopening this as I would like to add the following two line to the next version update:

```
    - python -c 'from minimalkv import get_store; get_store("fs", path=".")'
    - python -c 'from minimalkv import get_store; get_store("hfs", path=".")'
```

`fsspec` should stay optional.

**Comment by xhochy** [comment]
Fixed by #17
